domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/RouteType.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity;
+
+import java.util.stream.Stream;
+
+public enum RouteType {
+    LIGHT_RAIL(0),
```

Comment:
I would use the exact descriptions in https://gtfs.org/reference/static#routestxt.

* `5` - `CABLE_TRAM`.
* `6` - `AERIAL_LIFT`

Also, this is missing `11` and `12` (see above link)

Reply:
Indeed, I am modifying this